### ISSUE TITLE
Critical type issues in `CrowdinApiClient\Model\LanguageTranslation`

### ISSUE THREAD

**Issue by christophedebatz** [crowdin/crowdin-api-client-php#184]
Hi,

We encountered a logic exception in `CrowdinApiClient\Model\LanguageTranslation`. The code seems to be ... wrong.

The constructor of your model is:

```
public function __construct(array $data = [])
    {
        parent::__construct($data);
        $this->stringId = (int)$this->getDataProperty('stringId');
        $this->contentType = (string)$this->getDataProperty('contentType');

        $this->translationId = $this->getDataProperty('translationId')
            ? (int)$this->getDataProperty('translationId') : null;
        $this->text = $this->getDataProperty('text')
            ? (string)$this->getDataProperty('text') : null;
        $this->plurals = $this->getDataProperty('plurals')
            ? (array)$this->getDataProperty('plurals') : null;
        $this->user = $this->getDataProperty('user')
            ? (array)$this->getDataProperty('user') : null;
    }
```

So you make all properties nullable (except stringId and contentType).
But, the properties and their getters are not nullable and typed in PHP:

```
public function getTranslationId(): int
    {
        return $this->translationId;
    }

    public function setTranslationId(int $translationId): void
    {
        $this->translationId = $translationId;
    }

    public function getText(): string
    {
        return $this->text;
    }

    public function getUser(): ?array
    {
        return $this->user;
    }

    public function setText(string $text): void
    {
        $this->text = $text;
    }

    public function getPlurals(): array
    {
        return $this->plurals;
    }

    public function setPlurals(array $plurals): void
    {
        $this->plurals = $plurals;
    }
```

So, in our code, we faced some runtime errors like
`Return value of CrowdinApiClient\Model\LanguageTranslation::getText() must be of the type string, null returned`

We will be forced to not use your SDK since it can throws illegal errors at runtime.
It's clearly a critical issue for whoever uses this sdk..

Can you please fix it soon?

Thanks

**Comment by andrii-bodnar** [comment]
Hi @christophedebatz, thanks for reporting this!

Would you like to submit a PR?

**Comment by andrii-bodnar** [comment]
@christophedebatz available in [1.13.1](https://github.com/crowdin/crowdin-api-client-php/releases/tag/1.13.1)
